### ISSUE TITLE
Error "WebSocket connection was closed and cannot be recovered due to identification issues"

### ISSUE THREAD

**Issue by MarvinKlar** [discord-jda/JDA#2555]
### General Troubleshooting

- [X] I have checked for similar issues on the Issue-tracker.
- [X] I have checked for PRs that might already address this issue.

### Version of JDA

4.3.0

### Expected Behaviour

It should not throw the following error:
```
[11:48:18] [JDA MainWS-WriteThread/ERROR]: [net.dv8tion.jda.internal.requests.WebSocketClient] WebSocket connection was closed and cannot be recovered due to identification issues
CloseCode(4014 / Disallowed intents. Your bot might not be eligible to request a privileged intent such as GUILD_PRESENCES or GUILD_MEMBERS.)
```

The bot dos have all required permissions, see: https://prnt.sc/NpFPZsJYS9C9

### Code Example for Reproduction Steps

```java
// I just created the bot. The error occoured directly afterwards.
```

### Code for JDABuilder or DefaultShardManagerBuilder used

```java
JDA jda = JDABuilder.create(token, GatewayIntent.GUILD_MESSAGE_REACTIONS, GatewayIntent.GUILD_MESSAGES)
                    .addEventListeners(this)
                    .setAutoReconnect(true)
                    .disableCache(CacheFlag.ACTIVITY, CacheFlag.VOICE_STATE, CacheFlag.EMOTE, CacheFlag.CLIENT_STATUS,
                            CacheFlag.ONLINE_STATUS)
                    .build();
```

### Exception or Error

```yesyes
[11:48:18] [JDA MainWS-WriteThread/ERROR]: [net.dv8tion.jda.internal.requests.WebSocketClient] WebSocket connection was closed and cannot be recovered due to identification issues
CloseCode(4014 / Disallowed intents. Your bot might not be eligible to request a privileged intent such as GUILD_PRESENCES or GUILD_MEMBERS.)
```

**Comment by MinnDevelopment** [comment]
You haven't granted the bot the required intent (which is not a permission). See the [troubleshooting guide](https://jda.wiki/using-jda/troubleshooting/#im-getting-closecode4014-disallowed-intents) for details.

**Comment by MarvinKlar** [comment]
As I wrote: `The bot does have all required permissions, see: https://prnt.sc/NpFPZsJYS9C9`

Or did I do anything wrong regarding to the screenshot?

**Comment by MarvinKlar** [comment]
@MinnDevelopment

**Comment by freya022** [comment]
Permissions and intents are not the same thing at all.
As the issue template says, you should have tried running that with a more recent version, no need to update the whole bot, just up to the part that throws the error.

Anyways, the issue is that you're running an older version, that enables the "message content" intent by default, so you need to enable it on your dashboard. Go follow the guide Minn sent and enable that intent.

**Comment by MinnDevelopment** [comment]
Try updating to the latest version. We are currently on 5.0.0-beta.15 and 4.3.0 is several years outdated and using an older version of the Discord API, which has been deprecated for a long time.

**Comment by MarvinKlar** [comment]
I already gave the bot the "message content" intent permission. Didn't I? https://prnt.sc/NpFPZsJYS9C9

**Comment by freya022** [comment]
Yes, you can check if this is the right bot you added the intents for, reset your token and use it to make sure, or update your bot

**Comment by MarvinKlar** [comment]
How do I update the bot?

**Comment by MarvinKlar** [comment]
Got it! Thanks for your help!
